### PR TITLE
Netlify ecommerce App - fixes

### PR DESCRIPTION
**Pull request by kathirsvn**
Netlify ecommerce App - Clean up of options to run the sample app against Astra Vs JSON API and updated README file as well.

### CODE REVIEW

**Review by Yuqi-Du — APPROVED** [review]
LGTM.
May switch all localhost to 127.0.0.1? Since all higher node version may have issue resolving localhost. But not necessary an item we should take care.
I added a troubleshooting in the repo readme before.

**Comment by kathirsvn** [comment]
Good point @Yuqi-Du I changed all the instances of `localhost` to `127.0.0.1` now.

File: netlify-functions-ecommerce/README.md
Comment (Yuqi-Du) [inline netlify-functions-ecommerce/README.md:1]: Small thing, AUTH_URL should be 8081 port.
Related files: README and .env.example

**Review by Yuqi-Du — CHANGES_REQUESTED** [review]

File: netlify-functions-ecommerce/README.md
Comment (kathirsvn) [reply netlify-functions-ecommerce/README.md:1]: Fixed in this commit https://github.com/stargate/stargate-mongoose-sample-apps/pull/23/commits/79176fa71b2634dd3fd2b399afdd6db68b704324.

File: netlify-functions-ecommerce/.env.example [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#Environment to choose. Possible values: astra, jsonapi
```
Comment (vkarpov15) [inline netlify-functions-ecommerce/.env.example:1]: I would advise against using non-standard values for NODE_ENV. So best to not rely on anything other than "local", "development", or "production" for NODE_ENV.

Better to just have an IS_ASTRA environment variable.

File: netlify-functions-ecommerce/.config/index.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-  } catch (error) {
-    module.exports = require('./development');
-  }
+  console.log('Using ' + process.env.NODE_ENV + ' profile');
+  module.exports = require('./' + process.env.NODE_ENV);
 } else {
-  console.log('using production');
-  module.exports = require('./production');
+  console.log('Using jsonapi profile' + process.env.NODE_ENV);
+  module.exports = require('./jsonapi');
```
Comment (vkarpov15) [inline netlify-functions-ecommerce/.config/index.js:10]: In general, I'd recommend getting rid of this file if we're going to use dotenv. Having a multi-layered config setup that mixes different config patterns for a sample app is unnecessarily complex; makes it harder for developers to understand what is going on.

**Review by vkarpov15 — COMMENTED** [review]
I made a couple of quick fixes. Most notably, calling `dotenv.config()` is necessary at the start of every possible entrypoint.

I gather that the config file pattern has caused some confusion, so I'm open to switching over to using dotenv since that's a more common pattern for sample apps. However, if we go the dotenv route, we should use dotenv correctly, and we shouldn't mix config patterns. Does that make sense?

**Comment by vkarpov15** [comment]
That works for me @kathirsvn :+1:

File: netlify-functions-ecommerce/README.md
Comment (kathirsvn) [reply netlify-functions-ecommerce/README.md:1]: Hi @Yuqi-Du I fixed this part. Can you please approve this PR if you completed your review?

**Review by vkarpov15 — APPROVED** [review]

**Review by Yuqi-Du — APPROVED** [review]

**Pull request merged**